packages/did-auth-siop-op-authenticator/package.json

```diff
@@ -6,17 +6,17 @@
   "types": "dist/index.d.ts",
   "veramo": {
     "pluginInterfaces": {
-      "IDidAuthSiopOpAuthenticator": "./src/types/IDidAuthSiopOpAuthenticator.ts"
+      "IDidAuthSiopOpAuthorizer": "./src/types/IDidAuthSiopOpAuthorizer.ts"
     }
   },
   "scripts": {
     "build": "tsc --build"
   },
   "dependencies": {
-    "@sphereon/did-auth-siop": "0.2.14",
+    "@sphereon/did-auth-siop": "0.3.0-unstable.1",
```

Comment:
Please use ^0.3.0. The previous fixation was to ensure it wouldn't start using any version with the updated APIs

Reply:
fixed

Reply:
When it is ready. Right now we have unstable versions, as the refactoring of did-auth-siop isn't fully finished yet. The biggest parts are done, but more work is needed.